gdal_subsetter/coordinate_utilities.py

```diff
@@ -0,0 +1,119 @@
+""" Utilities for handling coordinate transformations and similar functionality
+    within the Harmony GDAL Adapter.
```

Comment:
Hopefully the division of shape file vs coordinate-stuff in the main granules makes sense. If not, I'm happy to shimmy functions around in a better set of modules.